testcases/open_posix_testsuite/conformance/interfaces/pthread_rwlock_unlock/4-1.c

```diff
@@ -31,7 +31,7 @@ int main(void)
 	static pthread_rwlock_t rwlock;
 	int rc;
 
-#ifdef __linux__
+#if defined(__linux__) || defined(__NuttX__)
 	printf("Unlocking uninitialized rwlock is undefined on Linux\n");
```

Comment:
I guess that we should remove the "on Linux" part from the message now, otherwise it looks fine.